### PR TITLE
User Orders Show page

### LINKED ISSUE #54: User Story 29, User views an Order Show Page
```
As a registered user
When I visit my Profile Orders page
And I click on a link for order's show page
My URL route is now something like "/profile/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item I ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order
```

DO THESE STORIES FIRST:
- User Story 28, User Profile displays Orders

### Checklist:

- [x] base tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #241

child of #274

depends on #239

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in the Default User Order Show page, and the necessary methods needed to present needed information.
The Order show page will provide all info that we would see on the Orders index, but with added sections for each included Item. These sections include name, description, image, price, and subtotal. This Subtotal required an instance method, which is called on an OrderItem. An OrderItem has the knowledge of quantity of items, and item price, which allows us to return the subtotal value.
This PR also refactored some display of Grand Totals on Orders as well, using the #number_to_currency method in ActiveView.

Resolves #54

### CODE REVIEW

File: spec/features/users/default/orders_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        expect(page).to have_content("#{@item_3.name}")
+        expect(page).to have_content("#{@item_3.description}")
+        expect(page).to have_css("img[src*='#{@item_3.image}']")
+        expect(page).to have_content("Quantity: #{@order_item_3.quantity}")
+        expect(page).to have_content("Subtotal: $#{@order_item_3.sub_total}")
+      end
+
+      expect(page).to have_content("Number of Items: #{@order_1.item_count}")
+      expect(page).to have_content("Grand Total: $#{@order_1.grand_total}")
+    end
```
Comment (VinceCarollo) [inline spec/features/users/default/orders_show_spec.rb:63]: Thanks for taking the time to test thoroughly!

**Review by VinceCarollo — APPROVED** [review]
